[debug] [2020-10-19T07:02:53.306Z] ----------------------------------------------------------------------
[debug] [2020-10-19T07:02:53.308Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\brownderek\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js projects:addfirebase todo-3534f
[debug] [2020-10-19T07:02:53.308Z] CLI Version:   8.12.1
[debug] [2020-10-19T07:02:53.308Z] Platform:      win32
[debug] [2020-10-19T07:02:53.309Z] Node Version:  v12.14.0
[debug] [2020-10-19T07:02:53.310Z] Time:          Mon Oct 19 2020 01:02:53 GMT-0600 (Mountain Daylight Time)
[debug] [2020-10-19T07:02:53.310Z] ----------------------------------------------------------------------
[debug] [2020-10-19T07:02:53.310Z] 
[debug] [2020-10-19T07:02:53.318Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-10-19T07:02:53.319Z] > authorizing via signed-in user
[debug] [2020-10-19T07:02:53.327Z] >>> HTTP REQUEST POST https://firebase.googleapis.com/v1beta1/projects/todo-3534f:addFirebase  
 
[debug] [2020-10-19T07:02:53.884Z] <<< HTTP RESPONSE 409 {"vary":"X-Origin, Referer, Origin,Accept-Encoding","content-type":"application/json; charset=UTF-8","date":"Mon, 19 Oct 2020 07:02:53 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-Q050=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-10-19T07:02:53.885Z] <<< HTTP RESPONSE BODY {"error":{"code":409,"message":"Requested entity already exists","status":"ALREADY_EXISTS"}}
[debug] [2020-10-19T07:02:53.887Z] HTTP Error: 409, Requested entity already exists
[debug] [2020-10-19T07:02:54.025Z] FirebaseError: HTTP Error: 409, Requested entity already exists
    at module.exports (C:\Users\brownderek\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:38:12)
    at Request._callback (C:\Users\brownderek\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:41:35)
    at Request.self.callback (C:\Users\brownderek\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at Request.emit (events.js:210:5)
    at Request.EventEmitter.emit (domain.js:475:20)
    at Request.<anonymous> (C:\Users\brownderek\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1154:10)
    at Request.emit (events.js:210:5)
    at Request.EventEmitter.emit (domain.js:475:20)
    at IncomingMessage.<anonymous> (C:\Users\brownderek\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1076:12)
    at Object.onceWrapper (events.js:299:28)
    at IncomingMessage.emit (events.js:215:7)
    at IncomingMessage.EventEmitter.emit (domain.js:475:20)
    at endReadableNT (_stream_readable.js:1184:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
[error] 
[error] Error: Failed to add Firebase to Google Cloud Platform project. See firebase-debug.log for more info.
